extensions/cloud/portability-cloud-google/config/common.yaml

```diff
@@ -0,0 +1,2 @@
+# Google cloud settings. See environments/ for environment-specific settings.
+cloud: GOOGLE
```

Comment:
Can we add a list of example options to the comment

Reply:
I think this is the only common setting left actually now that encryptionFlow and serviceProviderClasses are deprecated